.mage/magefile.go

```diff
@@ -25,15 +27,36 @@ import (
 	"go.einride.tech/mage-tools/targets/mggoreview"
 )
 
+func init() {
+	mgmake.AddExtraMakefiles(
+		[]mgmake.Makefile{
+			{
+				Path:          mgtool.GetGitRootPath("proto/Makefile"),
```

Comment:
spontaneous naming thought - `mgpath.FromGitRoot`?

Reply:
Ah, I like it!